### ISSUE TITLE
Black screen

### ISSUE THREAD

**Issue by RandyAndy1** [ioBroker/ioBroker.fullcalendar#284]
**Describe the bug**  
I am testing currently the Version 2.4.2. But clicking the Kalender only a back screen appears.
![grafik](https://github.com/ioBroker/ioBroker.fullcalendar/assets/46190910/df31de56-cd27-45ff-bdc3-0822843d9582)

**To Reproduce**  
1. Selet Admin
2. Click on KAlender
3. No editiong possible

**Versions:**  
 - Adapter version: 2.42
 - JS-Controller version: 6.06 (and I know it is not the satbale version)
 - Node version: 18.20.3
 - Operating system: Linux (Raspberry Pi 4)

**Additional context**

**Comment by kago76** [comment]
Same here: blank screen.

- Adapter version: 2.42
- js-controller version: 6.07
- Node version: 20.15.1
- OS: debian 12

**Comment by RandyAndy1** [comment]
IN Version 1.7.3 everything the black screen does not appear.

**Comment by PietPinguin** [comment]
blank screen too. 

Adapter: 2.42
Admin: 7.0.23
Node: 20.17.0
js-controller: 6.09
OS: Debian 11

**Comment by Gerri1** [comment]
blank screen too.

Adapter: 2.42
Admin: 7.0.23
Node: 18.20.4
js-controller: 6.09
OS: Debian 10

**Comment by pek24** [comment]
Same for me
Adapter: 2.4.2
Admin: 7.0.23
Node: 18.17.1
js-controller: 6.11
OS: Debian 11
I get always this log entries if I try open settings:
admin.0	2024-09-02 10:36:40.257	info	<== Disconnect system.user.admin from ::ffff:192.xxx.xxx.xxx fullcalendar
admin.0	2024-09-02 10:36:28.171	info	==> Connected system.user.admin from ::ffff:192.xxx.xxx.xxx

**Comment by Apollon77** [comment]
Are there any errors in the browser console?

**Comment by RandyAndy1** [comment]
Will check. Since I am traveling right now I can do that earliest next weekend.
Maybe the other reporter are faster and can check.

**Comment by kago76** [comment]
Same here:
Adapter: 2.4.2
js-controller: 6.0.11
Admin: 7.1.0
Node 20.17.0

I get two errors in the browser console:

```
TypeError: Cannot read properties of undefined (reading 'palette')
    at iv (Calendar.js:423:26)
    at gr (react-dom.production.min.js:160:137)
    at Si (react-dom.production.min.js:289:337)
    at xl (react-dom.production.min.js:279:389)
    at zl (react-dom.production.min.js:279:320)
    at gl (react-dom.production.min.js:279:180)
    at rl (react-dom.production.min.js:270:88)
    at ol (react-dom.production.min.js:267:429)
    at y (scheduler.production.min.js:13:203)
    at MessagePort.L (scheduler.production.min.js:14:128)
```

```
Calendar.js:423 Uncaught TypeError: Cannot read properties of undefined (reading 'palette')
    at iv (Calendar.js:423:26)
    at gr (react-dom.production.min.js:160:137)
    at Si (react-dom.production.min.js:289:337)
    at xl (react-dom.production.min.js:279:389)
    at zl (react-dom.production.min.js:279:320)
    at gl (react-dom.production.min.js:279:180)
    at rl (react-dom.production.min.js:270:88)
    at ol (react-dom.production.min.js:267:429)
    at y (scheduler.production.min.js:13:203)
    at MessagePort.L (scheduler.production.min.js:14:128)
```

**Comment by klein0r** [comment]
Same here.

```
 undefined is not an object (evaluating 't.theme.palette')
```

**Comment by bubiman** [comment]
Same here:
Plattform: docker
Node.js: v20.17.0 (Empfohlene Version v18.20.4)
NPM: 10.8.2
Adapter: 2.4.2

**Comment by kago76** [comment]
Problem is fixed on my system with version 2.4.5.
Thank you!
